utils/automation/fossologyscanner.py

```diff
@@ -48,6 +54,11 @@ def get_ci_name():
     RUNNING_ON = 'TRAVIS'
     TRAVIS_REPO_SLUG = os.environ['TRAVIS_REPO_SLUG']
     TRAVIS_PULL_REQUEST = os.environ['TRAVIS_PULL_REQUEST']
+  elif 'GITHUB_ACTIONS' in os.environ and os.environ['GITHUB_ACTIONS'] == 'true':
+    RUNNING_ON = 'GITHUB'
+    GITHUB_TOKEN = os.environ['GITHUB_TOKEN']
+    GITHUB_REPOSITORY = os.environ['GITHUB_REPOSITORY']
+    GITHUB_PULL_REQUEST = os.environ['GITHUB_PULL_REQUEST']
```

Comment:
Unable to find documentation for `GITHUB_PULL_REQUEST`. Can you please share the same?

Reply:
Got it from the description. It is coming from `-e GITHUB_PULL_REQUEST=${{ github.event.number }}`